tsconfig.spec.json

```diff
@@ -1,11 +1,11 @@
 {
   "extends": "./tsconfig.json",
   "include": ["./jest/testing-library__jest-dom.d.ts", "./jest/verifyComponent.d.ts", "**/*.spec.tsx", "**/*.spec.ts"],
-  "exclude": [],
+  "exclude": ["cypress"],
   "compilerOptions": {
     "declaration": false,
     "declarationMap": false,
-    "noImplicitReturns": true,
+    "noImplicitReturns": false,
```

Comment:
I'm not sure why this is necessary. Leaving it to `true` cause the `spec` typecheck to find return errors in source files which is confusing. There's a separate type check process for source files. I also don't think implicit return checks are necessary for Jest spec files. You can see the TS config file here is more lax than the config for source files. I don't consider jumping through extra hoops to keep the type system happy is productive in spec files.